xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPBaseRenderer.cpp

```diff
@@ -7,7 +7,11 @@
  */
 
 #include "RPBaseRenderer.h"
-
+#include "ServiceBroker.h"
```

Comment:
A design constraint is that ServiceBroker.h isn't allowed in this file. See how other singletons are handled.

Reply:
Ok will take care of this.